lib/realtime_web/channels/auth/channels_authorization.ex

```diff
@@ -5,17 +5,17 @@ defmodule RealtimeWeb.ChannelsAuthorization do
   require Logger
   import Realtime.Logs
 
+  @doc """
+  Authorize connection to access channel
+  """
+  @spec authorize(binary(), binary(), binary() | nil) :: {:ok, map()} | {:error, any()}
   def authorize(token, jwt_secret, jwt_jwks) when is_binary(token) do
     token
     |> clean_token()
     |> RealtimeWeb.JwtVerification.verify(jwt_secret, jwt_jwks)
   end
 
-  def authorize(_token, _jwt_secret, _jwt_jwks), do: :error
-
-  defp clean_token(token) do
-    Regex.replace(~r/\s|\n/, URI.decode(token), "")
-  end
```

Comment:
are you sure? this function is used in the authorize/3 function

Reply:
moved `clean_token/1` it to the end of the file